docs/install/collecting-metrics/README.md

```diff
@@ -0,0 +1,106 @@
+   [install from these instructions](https://cert-manager.io/docs/installation/kubernetes/).
+
+1. The first step is to install the operator:
+
+   ```shell
+   kubectl apply --filename https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
+   ```
+
+1. Next, create a namespace for the collector to run in:
+
+   ```shell
+   kubectl create namespace metrics
+   ```
+
+1. And then create an OpenTelemetryCollector object to store the configuration
+   for the collector:
+
+   ```shell
+   kubectl apply --filename collector.yaml
+   ```
+
+1. Finally, update the `config-observability` ConfigMap in Knative Serving and
+   Eventing
+   ```shell
+   kubectl patch --namespace knative-serving configmap/config-observability \
+     --type merge \
+     --patch '{"data":{"metrics.backend-destination":"opencensus","metrics.opencensus-address":"otel-collector.metrics:55678"}}'
+   kubectl patch --namespace knative-eventing configmap/config-observability \
+     --type merge \
+     --patch '{"data":{"metrics.backend-destination":"opencensus","metrics.opencensus-address":"otel-collector.metrics:55678"}}'
+   ```
+
+You can check that metrics are being forwarded by loading the Prometheus export
+port on the collector:
+
+```shell
+kubectl port-forward --namespace metrics deployment/otel-collector 8889
+```
+
+And then fetch http://localhost:8889/metrics to see the exported metrics.
+
+## Setting up Prometheus
+
+[Prometheus](https://prometheus.io/) is an open-source tool for collecting and
+aggregating timeseries metrics. Full configuration of Prometheus can be found at
+the website, but this document will provide a simple setup for scraping the
+OpenTelemetry Collector we set up in the previous section.
+
+1. Install the
+   [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator).
+   Note that the provided manifest installs the operator into the `default`
+   namespace. If you want to install into another namespace, you'll need to
+   download the YAML manifest and update all the namespace references to your
+   target namespace.
+
+   ```shell
+   kubectl apply --filename https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/bundle.yaml
+   ```
+
+1. You'll then need to set up a ServiceMonitor object to track the OpenTelemetry
+   Collector, as well as a ServiceAccount with the ability to read Kubernetes
+   services and pods (so that Prometheus can track the resource endpoints) and
+   finally a Prometheus object to instantiate the actual Prometheus instance.
+
+   ```shell
+   kubectl apply --filename prometheus.yaml
+   ```
+
+By default, the Prometheus instance will only be exposed on a private service
+named `prometheus-operated`; to access the console in your web browser, run:
+
+```shell
+kubectl port-forward --namespace metrics service/prometheus-operated 9090
+```
+
+And then access the console in your browser via http://localhost:9090.
```

Comment:
What should we do for dashboard like grafana? should we provide them or not? Or that could be a different doc

Reply:
I feel like we should document each component the available metric name and a short description. 
Maybe like a table format that includes component, metric, description

Reply:
I think this is a subsequent (post 0.19) work item to document the exported metrics (possibly rationalizing them).

We absolutely should do this, and I expect that we'd need to treat it like other API contracts. I wasn't quite ready to do that here, but I can give 1 or 2 sample queries.

Reply:
I think it would be really useful to have a doc listing and rationalizing the exported metrics.

I suspect we'll find a lot that don't warrant being exported by default.

As you suggested, I think it will be useful to have metric for this document that can act as a litmus test, so folks who aren't up to speed on metrics know what they're looking for to gauge success.

Reply:
I'm not sure about not exporting them by default -- the cost of an exported metric should not be particularly high, but I think we *do* need to rationalize both the metric names and labels.

Reply:
Next step should be what metric matters to check first in certain scenarios eg. debugging.